build-apple/Sweetspace.xcodeproj/project.pbxproj

```diff
@@ -127,6 +127,7 @@
 /* End PBXContainerItemProxy section */
```

Comment:
Didn't know if this file needs to be added so I included it, but I can be deleted if not needed.